OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -47,17 +48,18 @@ public class GravityBomb : IProjectile, ISync
 		readonly GravityBombInfo info;
 		readonly Animation anim;
 		readonly ProjectileArgs args;
+		readonly WVec acceleration;
 		[Sync] WVec velocity;
 		[Sync] WPos pos;
-		[Sync] WVec acceleration;
 
 		public GravityBomb(GravityBombInfo info, ProjectileArgs args)
 		{
 			this.info = info;
 			this.args = args;
 			pos = args.Source;
-			velocity = new WVec(WDist.Zero, WDist.Zero, -info.Speed);
-			acceleration = new WVec(WDist.Zero, WDist.Zero, info.Acceleration);
+			var facing = (args.PassiveTarget - pos).Yaw.Facing;
+			velocity = -info.Velocity.Rotate(WRot.FromFacing(facing));
```

Comment:
> which will then also use the facing from the firing actor (which is IMO more appropriate than the source-target vector). 

I disagree for once, we now support sideway projectile movement so hardcoding it to source facing instead of facing the target feels wrong to me.